homeassistant/components/airvisual/strings.json

```diff
@@ -18,6 +18,13 @@
           "password": "[%key:common::config_flow::data::password%]"
         }
       },
+      "reauth_confirm": {
+        "title": "Re-authenticate AirVisual",
```

Comment:
Do we not have a re-auth title common string yet ?

Reply:
We do not. Should I add one as part of this PR?

Reply:
No, in a new PR.